megamek/src/megamek/common/battlevalue/BVCalculator.java

```diff
@@ -711,7 +711,7 @@ protected double arcFactor(Mounted equipment) {
     }
 
     /**
-     * Forwards to {@link #processWeapon(Mounted, boolean, boolean, int)} with a weaponCount
+     * Forwards to {@link #maintainUsedSearchlight(Mounted, boolean, boolean, int)} with a weaponCount
```

Comment:
???

Reply:
No idea; it appears to be an errant s-n-r from a prior patch.
Reverted.